content/03-originate/originate-sdk.mdx

```diff
@@ -130,60 +142,62 @@ assert(owner.did === alice.did);
+So when we wrote `await` in `await apiAlice.getSigningIdentity()`, this is no accident. We indeed need to do a round trip to the blockchain to know what account our public key is associated with.
 
 </HighlightBox>
 
 ## Ticker creation
 
-Now that the ticker is reserved, it is time to issue the security token.
+Now that the ticker is reserved, it is time to issue the asset.
 
 Oh wait! The reservation may have happened some time ago. After all a reservation remains valid for 60 days, for instance. And your `const reservation: TickerReservation` instance might no longer be in memory.
 
-How do you get it back? Use [`getTickerReservation`](https://developers.polymath.network/polymesh-sdk-api-reference/classes/polymesh#gettickerreservation):
+How do you get it back? Use [`assets.getTickerReservation`](https://github.com/PolymathNetwork/polymesh-sdk/wiki/api.client.Assets.Assets#gettickerreservation):
 
 <MultiCodeBlock>
 
 ```typescript
+const signingManagerAlice: LocalSigningManager = await LocalSigningManager.create({...});
 const apiAlice: Polymesh = await Polymesh.connect({...});
-const reservation: TickerReservation = await apiAlice.getTickerReservation({
+const reservation: TickerReservation = await apiAlice.assets.getTickerReservation({
     "ticker": "ACME"
 });
 ```
 
 ```javascript
+const signingManagerAlice = await LocalSigningManager.create({...});
 const apiAlice = await Polymesh.connect({...});
-const reservation = await apiAlice.getTickerReservation({
+const reservation = await apiAlice.assets.getTickerReservation({
     "ticker": "ACME"
 });
 ```
 
 </MultiCodeBlock>
 
-You will note that the class `TickerReservation`, just like the transaction queue, keeps a [`protected context: Context`](https://developers.polymath.network/polymesh-sdk-api-reference/classes/tickerreservation#properties-1). This `context` in turn keeps a [`polymeshApi`](https://developers.polymath.network/polymesh-sdk-api-reference/classes/context#properties-1). It is in there that we find the implicit knowledge that it is `alice`'s account that is asking for the next actions. If it were any other **account** that had called `getTickerReservation`, this other account would not be able to follow up with a `.createToken` command because it doesn't own the reservation.
+You will note that the class `TickerReservation`, just like the transaction queue, keeps a [`protected context: Context`](https://developers.polymath.network/polymesh-sdk-api-reference/classes/tickerreservation#properties-1). This `context` in turn keeps a [`polymeshApi`](https://developers.polymath.network/polymesh-sdk-api-reference/classes/context#properties-1). It is in there that we find the implicit knowledge that it is `alice`'s account that is asking for the next actions. If it were any other **account** that had called `assets.getTickerReservation`, this other account would not be able to follow up with a `.createAsset` command because it doesn't own the reservation.
```

Comment:
Should we be explaining the functionality of `Context` here, since we are not even exporting it in docs?

Reply:
Thoughts @VictorVicente @polymath-eric ?

Reply:
Lets leave it for now and since v16 is very much around the corner lets update is all together once TQs are also removed